maltoolbox/model.py

```diff
@@ -2,58 +2,61 @@
 MAL-Toolbox Model Module
 """
 
+from __future__ import annotations
+from dataclasses import dataclass, field
 import json
 import logging
+from typing import Any, Dict, List, Optional, Set, Tuple
 
-from dataclasses import dataclass
+from maltoolbox.language import LanguageClassesFactory
 from maltoolbox.file_utils import (
     load_dict_from_json_file, load_dict_from_yaml_file,
     save_dict_to_file
 )
 
-from .__init__ import __version__
+from maltoolbox import __version__
 
 logger = logging.getLogger(__name__)
 
 @dataclass
 class AttackerAttachment:
     """Used to attach attackers to attack step entrypoints of assets"""
-    id: int = None
-    name: str = None
-    entry_points: list[tuple] = None
+    id: Optional[int] = None
+    name: Optional[str] = None
+    entry_points: list[tuple] = field(default_factory=lambda: [])
```

Comment:
Does it make sense to have an attacker attachment that has these properties to `None/[]`? It seems to me no.

Reply:
Agreed, I think this is the case in some other places as well (decide what should be optional and not)

Reply:
Should we track this discussion here or in a new issue?

Reply:
Tracking it here instead: #59